src/cookbooks/looker/performance_caching.md

```diff
@@ -0,0 +1,75 @@
+# Optimizing Looker Performance - Caching
+
+This how-to guide is about improving performance of Looker dashboards and explores using caching. This is particularly
+useful if you have many users accessing a dashboard or explore, especially across timezones.
+
+This guide has two alternatives: Applying Generated Datagroups and Custom Datagroups
+
+> ⚠️ Some steps in this guide require knowledge of LookML and a developer license - ask in #data-help for assistance if
```

Comment:
Shall we add the link to request a license to make it easier and reduce the data-help messages?

> ⚠️ Some steps in this guide require a Looker developer license. To request a license, please submit a [Request to the Data Org in Jira](https://mozilla-hub.atlassian.net/wiki/spaces/DATA/pages/6849444). For additional assistance, including LookML code, use the #data-help channel in Slack.